34th-SOPT-Seminar/LoginViewController.swift

```diff
@@ -9,39 +9,61 @@ import UIKit
 
 class LoginViewController: UIViewController {
 
+    private lazy var customInputAccessoryView: UIView = {
+        let bar = UIToolbar(frame: CGRect(origin: .zero, size: CGSize(width: 100, height: 100)))
+        let hideKeyboardButton = UIBarButtonItem(
+            image: UIImage(systemName: "keyboard.chevron.compact.down"),
+            style: .plain,
+            target: self,
+            action: #selector(keyboardHideButtonTapped)
+        )
+        hideKeyboardButton.tintColor = UIColor.dangGeunOrange
+        let flexibleBarButton = UIBarButtonItem(systemItem: UIBarButtonItem.SystemItem.flexibleSpace)
+        bar.items = [flexibleBarButton, hideKeyboardButton]
+        bar.sizeToFit()
+        return bar
+    }()
+    
     private let titleLabel: UILabel = {
         let label = UILabel(frame: CGRect(x: 69, y: 161, width: 236, height: 44))
         label.text = "동네라서 가능한 모든것\n당근에서 가까운 이웃과 함께해요."
         label.textColor = .black
         label.textAlignment = .center
         label.numberOfLines = 2
-        label.font = UIFont.systemFont(ofSize: 18, weight: .bold)
+        label.font = UIFont.pretendardFont(ofSize: 18, weight: 700) //bold
         label.clipsToBounds = true
         return label
     }()
 
     private lazy var idTextField: UITextField = {
         let textField = UITextField(frame: CGRect(x: 20, y: 276, width: 335, height: 52))
         textField.placeholder = "아이디를 입력해주세요"
-        textField.font = UIFont.systemFont(ofSize: 18, weight: .semibold)
+        textField.font = UIFont.pretendardFont(ofSize: 18, weight: 600) //semiBold
         textField.backgroundColor = UIColor(red: 221/255, green: 222/255, blue: 227/255, alpha: 1)
+        textField.borderStyle = .roundedRect
+        textField.inputAccessoryView = self.customInputAccessoryView
         return textField
     }()
 
     private lazy var passwordTextField: UITextField = {
         let textField = UITextField(frame: CGRect(x: 20, y: 335, width: 335, height: 52))
         textField.placeholder = "비밀번호를 입력해주세요"
-        textField.font = UIFont.systemFont(ofSize: 14, weight: .semibold)
+        textField.font = UIFont.pretendardFont(ofSize: 14, weight: 600) //semiBold
         textField.backgroundColor = UIColor(red: 221/255, green: 222/255, blue: 227/255, alpha: 1)
+        textField.borderStyle = .roundedRect
+        textField.inputAccessoryView = self.customInputAccessoryView
         return textField
     }()
 
     private lazy var loginButton: UIButton = {
         let button = UIButton(frame: CGRect(x: 21, y: 422, width: 332, height: 58))
-        button.backgroundColor = UIColor(red: 255/255, green: 111/255, blue: 15/255, alpha: 1)
+        button.backgroundColor = UIColor.dangGeunOrange
```

Comment:
```suggestion
        button.backgroundColor = .dangGeunOrange
```
이렇게만 써도 되는것으로 알고있습니다!

Reply:
맞습니다!
UIColor을 생략하면 코드가 훨씬 간결해지죠!~👍
리뷰 감사합니당 :)